### ISSUE TITLE
Testing molecules against other fungal skinNTDs

### ISSUE THREAD

**Issue by wwjvdsande** [OpenSourceMycetoma/Series-1-Fenarimols#88]
From the sporotrichosis community I got a request which molecules would be great to screen against sporotrichosis too. I would suggest the fenarimols as a first choice as we have plenty synthesized. Would USyd and UCL be willing to share their molecules for that puprose too?

**Comment by mattodd** [comment]
All open, provided we're cited. Hopefully you have samples to share, otherwise delivery of fresh stocks would be the limiting factor, I suspect!

**Comment by bebi78** [comment]
@wwjvdsande Of course, I can also provide some active compounds for testing (e.g., niclosamides, naphthazarin, tyrphostin a9, cinnamaldehyde).
